anesthetic/plot.py

```diff
@@ -281,6 +293,14 @@ def fastkde_plot_1d(ax, data, *args, **kwargs):
 
     ans = ax.plot(x[i], p[i], color=color, *args, **kwargs)
     ax.set_xlim(*check_bounds(x[i], xmin, xmax), auto=True)
+
+    if filled1d is True:
+        c = iso_probability_contours_from_samples(p[i], contours=levels)
```

Comment:
I think that fastkde requires `iso_probability_contours` rather than `iso_probability_contours_from_samples`, since fastkde uses a grid, rather than samples (you can see this difference in between the 2d contour versions.

Reply:
Ahh sorry I missed this. It's an easy fix and I have double checked it works locally, I will push it shortly.